wiki/Client/Menu_tips/zh.md

```diff
@@ -82,7 +82,7 @@ tags:
 - 查阅[精选艺术家列表](https://osu.ppy.sh/beatmaps/artists)来获取已经测好时间轴的谱面，并直接开始作图！
 - 把哨、镲和拍手音效放在节奏对应的位置上来润色你的谱面！可以在 `时间轴设定 (Timing Setup)` 窗口 (`F6`) 内的音频 (Audio) 标签页下来修改不同段落的音效组。
 - 将光标悬停在时间轴上，并按住 `Alt`，滚动鼠标滚轮或者按 `+`/`-` 按钮来缩放时间轴。
-- 如果你在作图期间调整时间轴，切记要使用[`重新对齐所有物件`](/wiki/Client/Beatmap_editor/Menu#special-commands.3)功能，重新将所有物件对齐在时间轴上。这个功能位于 `测定时间轴 (Timing)` 选项卡的下拉菜单里。注意未对齐的[继承时间点（绿线）](/wiki/Client/Beatmap_editor/Timing#继承时间点（绿线）)——它们不受这个功能影响！
+- 如果你在作图期间调整时间轴，切记要使用[`全部重新对齐`](/wiki/Client/Beatmap_editor/Menu#测定时间轴-(timing))功能，重新将所有物件对齐在时间轴上。这个功能位于 `测定时间轴 (Timing)` 选项卡的下拉菜单里。注意未对齐的[继承时间点（绿线）](/wiki/Client/Beatmap_editor/Timing#继承时间点（绿线）)——它们不受这个功能影响！
```

Comment:
this is not the editor tip of the localisation in the game, and this refers to the button that does not exist, I only changed the name of the button that it's in game